src/unity/python/turicreate/toolkits/drawing_recognition/drawing_recognition.py

```diff
@@ -103,10 +106,21 @@ def update_progress(cur_loss, iteration):
                  epochs=num_epochs,
                  iterations=None)
 
+    ctx = _mxnet_utils.get_mxnet_context(max_devices=batch_size)
     model = _Model(num_classes = len(classes))
+    model_params = model.collect_params()
+    model_params.initialize(_mx.init.Xavier(), ctx=ctx)
+    if use_pretrained_model:
+        pretrained_model_url = _os.environ["PRETRAINED_MODEL_URL"]
+        temp_filename = _mkstemp()[1]
+        _six.moves.urllib.request.urlretrieve(pretrained_model_url,
+            temp_filename)
+        model.load_params(
+            temp_filename,
+            ctx=ctx,
+            allow_missing=True)
+        _os.unlink(temp_filename)
     softmax_cross_entropy = _mx.gluon.loss.SoftmaxCrossEntropyLoss()
-    
-    ctx = _mxnet_utils.get_mxnet_context(max_devices=batch_size)
 
     model_params = model.collect_params()
     model_params.initialize(_mx.init.Xavier(), ctx=ctx)
```

Comment:
Doesn't this overwrite the model params defined above?

Reply:
Interestingly, mxnet ignores reinitialization until the `force_reinit` flag is passed, so the functionality is unaffected by these lines. However, this code is dead for all practical purposes and shouldn't be checked in so I pushed an update to fix it.